### PR TITLE
Add ROBOT diff github actions

### PR DESCRIPTION
**Pull request by shawntanzk**
Related to https://github.com/INCATools/ontology-development-kit/issues/524
Notes: currently tests only -edit and main file, future enhancement should allow selection of files that this runs on

### CODE REVIEW

**Comment by shawntanzk** [comment]
think it is trying to replace ${{secrets.GITHUB_TOKEN}} with something, not sure how to get it to place that as is

**Comment by shawntanzk** [comment]
nvm - got it, will make the changes :)

File: template/_dynamic_files.jinja2 [hunk 1]
```diff
@@ -663,6 +663,125 @@ jobs:
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download reasoned diff
+        uses: actions/download-artifact@v2
+        with:
+          name: classification-diff.md
+          path: classification-diff.md
+      - name: Prepare reasoned comment
+        run: "echo \"Here's a diff of how these changes impact the classified ontology: \n\" >comment.md; cat classification-diff.md/classification-diff.md >>comment.md"
```
[the remark this answers is not in the document]
Comment (shawntanzk) [reply template/_dynamic_files.jinja2:761]: thats a good idea, not 100% sure if I know how to do this with this commenting system, but will give it a try

**Comment by matentzn** [comment]
@gouttegd you can merge this, was not sure if you were happy!

**Comment by gouttegd** [comment]
Oh yeah, sorry, I forgot I “requested changes” when I reviewed that  2 days ago. My bad, approving now!

**Review by gouttegd — APPROVED** [review]

**Pull request merged**